bot/handlers/sending_videos/manual_clip_handler.py

```diff
@@ -54,6 +56,11 @@ async def _do_handle(self, message: Message) -> None:
         if end_seconds <= start_seconds:
             return await self.__reply_end_time_earlier_than_start(message)
 
+        clip_duration = end_seconds - start_seconds
+        if not await DatabaseManager.is_admin_or_moderator(message.from_user.id) and clip_duration > settings.MAX_CLIP_DURATION:
+            await self.__answer_markdown(message, get_limit_exceeded_clip_duration_message())
+            return
```

Comment:
wydaje mi sie ze mam de ja vu: ten if juz gdzies jest, seems like mozna by cos z tym wyniesc

Reply:
no pewnie we wszystkich handlerach które wysyłają wideo tylko nwm jak to opakować bo do klasy nadrzędnej to nie dam to co funkcja jakąś jebnac i do utilsów ? tylko jak ja wtedy nazwać i czy będzie wtedy czytelniej

Reply:
a w sumie czemu nie do klasy bazowej xd

Reply:
`handle_clip_duration_limit_exceeded`

Reply:
> a w sumie czemu nie do klasy bazowej xd

a w klasie bazowej nie powinno być to co się tyczy wszystkich dzieci czyli tutaj wszystki handlerów ?

Reply:
To co mówisz tyczy się interfejsów/funkcji wirtualnych. Nie powinieneś zmuszać do implementacji czegoś niepotrzebnego, ale jak jest w klasie bazowej zaimplementowane to imo może sobie być. Nie ma przymusu używania wszystkich funkcji klasy bazowej.

Reply:
O_o nie ma ? szczerze to ja całe życie myślałem że na tym polega ta idea że klasa bazowa = część wspólna